source/nanoFramework.Json/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="mscorlib" publicKeyToken="c07d481e9758c731" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-1.7.3.5" newVersion="1.7.3.5" />
```

Comment:
app.config files are bad, it shows that there is an issue with the nugets installed. Try removing the file and ensure it still builds after updating the nugets.